modules/features/filters/src/main/java/au/com/shiftyjelly/pocketcasts/filters/FilterEpisodeListFragment.kt

```diff
@@ -92,7 +95,28 @@ class FilterEpisodeListFragment : BaseFragment() {
 
     private lateinit var imageLoader: PodcastImageLoader
 
-    private lateinit var adapter: EpisodeListAdapter
+    private val adapter: EpisodeListAdapter by lazy {
+        EpisodeListAdapter(
+            downloadManager = downloadManager,
+            playbackManager = playbackManager,
+            upNextQueue = upNextQueue,
+            settings = settings,
+            onRowClick = this::onRowClick,
+            playButtonListener = playButtonListener,
+            imageLoader = imageLoader,
+            multiSelectHelper = multiSelectHelper,
+            fragmentManager = childFragmentManager,
+            swipeButtonLayoutFactory = SwipeButtonLayoutFactory(
+                swipeButtonLayoutViewModel = swipeButtonLayoutViewModel,
+                onItemUpdated = this::lazyNotifyAdapterChanged,
+                defaultUpNextSwipeAction = { settings.getUpNextSwipeAction() },
+                context = requireContext(),
+                fragmentManager = parentFragmentManager,
+                swipeSource = EpisodeItemTouchHelper.SwipeSource.FILTERS,
+            )
+        )
+    }
```

Comment:
I moved the adapter initialization to be lazy because initializing it in `onAttach` was causing problems due to the fact that the `swibeButtonLayoutViewModel` had not been injected yet. Let me know if you think this might cause any problems.